code/modules/clothing/suits/ego_gear/he.dm

```diff
@@ -302,7 +304,7 @@ Any attempt to code risk class armor will result in a 10 day Github ban.*/
 	desc = "Even still, I witnessed man and sky and earth tear into thousands of pieces."
 	icon_state = "impending_day"
 	flags_inv = NONE
-	armor = list(RED_DAMAGE = 20, WHITE_DAMAGE = -20, BLACK_DAMAGE = 50, PALE_DAMAGE = 20) // 70
+	armor = list(RED_DAMAGE = 20, WHITE_DAMAGE = -20, BLACK_DAMAGE = 50, PALE_DAMAGE = 20, FIRE = 40) // 70
```

Comment:
Why is the fire resistance of this armor double the physical resistance if by default most armor has 50% of its physical resistance as fire resistance.
Shouldn't this be FIRE = 10?

Reply:
The normal half is calculated at initialize for most armors. Impending day gets a bonus because the abnormality itself will deal fire damage.